app.rb

```diff
@@ -12,11 +15,13 @@ def initialize
     @books = []
     @people = []
     @rentals = []
+    load_data
```

Comment:
- [ ] Please make sure that the app can function properly with no errors shown if there are one or more missing files, as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_preserve_data.md#:~:text=Remember%20to%20handle%20the%20case%20when%20one%20or%20more%20of%20the%20files%20are%20missing) requirement.

![image](https://user-images.githubusercontent.com/62290545/212199646-475eb406-da08-4e84-a9c6-7a3bf6482c75.png)

Reply:
- Created a condition for when the json file doesn't exist. 😄